.github/workflows/macos-deploy.yml

```
@@ -0,0 +1,92 @@
+          INSTALLER_CERT: "${{ secrets.OSX_INSTALLER_CERT }}"
+          SAFARI_PROVISION: "${{ secrets.OSX_SAFARI_PROVISION }}"
+          EXTENSION_PROVISION: "${{ secrets.OSX_EXTENSION_PROVISION }}"
+          MATCH_GIT_URL: "${{ secrets.OSX_MATCH_GIT_URL }}"
+          MATCH_PASSWORD: "${{ secrets.OSX_MATCH_PASSWORD }}"
+          GIT_AUTHORIZATION: "${{ secrets.OSX_GIT_AUTHORIZATION }}"
+          API_KEY: "${{ secrets.OSX_API_KEY }}"
+          API_KEY_ID: "${{ secrets.OSX_API_KEY_ID }}"
+          API_ISSUER_ID: "${{ secrets.OSX_API_ISSUER_ID }}"
+
+      - name: Upload Enkrypt pkg
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)